src/structures/Guild.js

```diff
@@ -534,6 +534,30 @@ class Guild extends AnonymousGuild {
     );
   }
 
+  /**
+   * Return maximum bitrate based on the guild premium_tiers level and the features
+   * @type {?number}
+   * @readonly
+   */
+  get maximumBitrate() {
+    if (this.features.includes('VIP_REGIONS')) {
+      return 384000;
+    } else {
+      switch (this.premiumTier) {
+        case PremiumTiers.indexOf(this.premiumTier) === 0:
+          return 96000;
+        case PremiumTiers.indexOf(this.premiumTier) === 1:
+          return 128000;
+        case PremiumTiers.indexOf(this.premiumTier) === 2:
+          return 256000;
+        case PremiumTiers.indexOf(this.premiumTier) === 3:
+          return 384000;
+        default:
+          return 96000;
```

Comment:
`PremiumTiers` is actually this:
```JavaScript
{
  '0': 'NONE',
  '1': 'TIER_1',
  '2': 'TIER_2',
  '3': 'TIER_3',
  NONE: 0,
  TIER_1: 1,
  TIER_2: 2,
  TIER_3: 3
}
```
The `.indexOf()` on this will cause a `TypeError`!

Reply:
Fixed. Damn, it would have been a big mistake.